src/sonic-frr/patch/0037-bgp-community-memory-leak-fix.patch

```diff
@@ -1,200 +1,7 @@
-From 139e450bccf3c2591c7b158662b36e5b9ff33172 Mon Sep 17 00:00:00 2001
+From 2b8e4e4b93a78e5884e2e5b97050b4ea3843e2e0 Mon Sep 17 00:00:00 2001
 From: Donald Sharp <sharpd@nvidia.com>
-Date: Sat, 2 Mar 2024 09:42:30 -0500
-Subject: [PATCH 1/5] bgpd: Include unsuppress-map as a valid outgoing policy
-
-If unsuppress-map is setup for outgoing peers, consider that
-policy is being applied as for RFC 8212.
-
-Signed-off-by: Donald Sharp <sharpd@nvidia.com>
----
- bgpd/bgp_route.c | 8 ++++----
- 1 file changed, 4 insertions(+), 4 deletions(-)
-
-diff --git a/bgpd/bgp_route.c b/bgpd/bgp_route.c
-index f4dff0d802..3a03644f93 100644
---- a/bgpd/bgp_route.c
-+++ b/bgpd/bgp_route.c
-@@ -5834,10 +5834,10 @@ bool bgp_outbound_policy_exists(struct peer *peer, struct bgp_filter *filter)
- 	if (peer->sort == BGP_PEER_IBGP)
- 		return true;
- 
--	if (peer->sort == BGP_PEER_EBGP
--	    && (ROUTE_MAP_OUT_NAME(filter) || PREFIX_LIST_OUT_NAME(filter)
--		|| FILTER_LIST_OUT_NAME(filter)
--		|| DISTRIBUTE_OUT_NAME(filter)))
-+	if (peer->sort == BGP_PEER_EBGP &&
-+	    (ROUTE_MAP_OUT_NAME(filter) || PREFIX_LIST_OUT_NAME(filter) ||
-+	     FILTER_LIST_OUT_NAME(filter) || DISTRIBUTE_OUT_NAME(filter) ||
-+	     UNSUPPRESS_MAP_NAME(filter)))
```

Comment:
Hi, @dgsudharsan , seems there was UNSUPPRESS_MAP_NAME(filter) in the first commit, but removed in the final version, is it expected? thanks.

Reply:
Hi @donaldsharp  can you please help answer this?

Reply:
it is missing, but the fix it came from `6814401c47b948a50` is a fix that is not related to the actual memory leak.  It is related to allowing a unsuppress-map as a valid outgoing policy.  Where it was not allowed before.